internal/audit_log_test.go

```diff
@@ -265,45 +270,61 @@ var _ = Describe("AuditLog Test", func() {
 		})
 
 		When("getting by user", func() {
+			// TODO: why aren't they the same? and is it a good idea to allow two users with the same email?
+			adminUser, err := userServiceClient().GetById(ctx, wrapperspb.String(testEnv.gatewayTestEnv.AdminUser.Id))
+			//adminUser, err := userServiceClient().GetByEmail(ctx, wrapperspb.String(testEnv.gatewayTestEnv.AdminUser.Email))
```

Comment:
The repository makes no assumptions if there is already another user with the same email. BUT when properly created via the user aggregate code in the command handler it would prevent creation of a second user with the same email address. This is more a bug in the test code because the actual system wouldn't behave like that.

Reply:
As discussed verbally I wrote a workaround for now and created https://github.com/finleap-connect/monoskope/issues/182 to deal with this issue. The workaround includes fixes for the tests so this PR should be ready for review again.